cloud/misc/proxmox_kvm.py

```diff
@@ -604,6 +604,19 @@
     name        : spynal
     node        : sabrewulf
     state       : current
+
+# Clone VM from template ( or another VM )
+- proxmox_kvm:
+       api_user    : tfm@pve
+       api_password: secret
+       api_host    : helldorado
+       clone       : 555
+       vmid        : 666
+       node        : proxmox2
+       name        : onetest
+       storage     : 'stor2'
+       format      : 'qcow2'
+       timeout     : 300
```

Comment:
The `timeout` for the task must have an default value like 1200 in assigned function see below.